src/types/index.ts

```diff
@@ -454,7 +454,6 @@ export interface IGAMetaTx {
 }
 
 export interface ITx {
-  abiVersion?: number;
```

Comment:
I'm planning to have working mdw types in sdk@14 🙏  so you can take these types from sdk